drivers/disk/usdhc.c

```diff
@@ -2822,6 +2832,11 @@ static int disk_usdhc_init(const struct device *dev)
 		    (DISK_ACCESS_USDHC_INIT_CD_PROPS(n)),		\
 		    (DISK_ACCESS_USDHC_INIT_NONE(n)))
 
+#define DISK_ACCESS_USDHC_INIT_NO_1_8_V(n)				\
+	COND_CODE_1(DT_INST_NODE_HAS_PROP(n, no_1_8_v),			\
```

Comment:
You don't need to check HAS_PROP for a boolean property. You can remove `DISK_ACCESS_USDHC_NO_1_8_V_PROPS` and `DISK_ACCESS_USDHC_INIT_NO_1_8_V`, and initialize the member unconditionally.

Reply:
Thanks, Done